util/timer/src/timer.rs

```diff
@@ -113,6 +120,15 @@ pub enum ScheduleError {
 }
 
 impl TimerApi {
+    // `mut` is not necessary. However it works as static protection
```

Comment:
What does `static protection` mean?

Reply:
Requiring `&mut self` even if the code has a internal mutability will discourage user from using the function after initialization.

Reply:
Removed it.